google/cloud/storage/client.py

```diff
@@ -953,6 +954,9 @@ def create_bucket(
         if location is not None:
             properties["location"] = location
 
+        if rpo is not None:
+            properties["rpo"] = rpo
+
```

Comment:
Is `rpo` "special" enough to warrant passing it to the constructor, rather than setting it afterward? `location` is an oddball, because it cannot be set once the bucket is created.  If that is true for `rpo` as well, then we shouldn't have a setter for the `rpo` property.  If `rpo` *can* be updated on an existing bucket, then we can just leave it out of the constructor params, ISTM.

Reply:
`rpo` can also be set after bucket creation.

Reply:
@tseaver i'm using this method as you suggested in the samples, but it seems .create is deprecated and will be removed in the future. Do you know if there's a different way we're supposed to be doing this?